apps/codesandbox-react-template/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "codesandbox-react-template",
+  "description": "Template for creating codesandbox environment for @fluenui/react* packages.",
+  "version": "7.0.0",
+  "private": true,
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "lint": "just-scripts lint",
+    "test": "just-scripts test",
+    "just": "just-scripts",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "start": "just-scripts dev"
+  },
+  "devDependencies": {
+    "@types/es6-promise": "0.0.32",
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@types/webpack-env": "1.15.1",
+    "@uifabric/tslint-rules": "^7.2.1",
+    "@uifabric/build": "^7.0.0"
+  },
+  "dependencies": {
+    "@uifabric/set-version": "^7.0.13",
+    "@microsoft/load-themed-styles": "^1.10.26",
+    "es6-promise": "^4.1.0",
+    "immutability-helper": "~2.8.1",
+    "@fluentui/react": "7.121.2",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "typescript": "3.7.2",
+    "tslib": "^1.10.0"
+  }
```

This looks like it was created with an outdated create-package template (or copy-pasted). As a starting point, the deps should be more like this. Once I understand more about the package purpose it will probably be possible to refine the list a bit more.
```suggestion
  "devDependencies": {
    "@types/react": "16.8.25",
    "@types/react-dom": "16.8.4",
    "@types/webpack-env": "1.15.1",
    "@uifabric/tslint-rules": "^7.2.1",
    "@uifabric/build": "^7.0.0"
  },
  "dependencies": {
    "@uifabric/set-version": "^7.0.13",
    "@microsoft/load-themed-styles": "^1.10.26",
    "@fluentui/react": "7.121.2",
    "react": "16.8.6",
    "react-app-polyfill": "~1.0.1",
    "react-dom": "16.8.6",
    "tslib": "^1.10.0"
  }
```